spec/tapioca/dsl/compilers/action_mailer_spec.rb

```diff
@@ -120,9 +120,7 @@ def notify_customer(customer_id); end
                 class NotifierMailer < ActionMailer::Base
                   delegate :notify_customer, to: :foo
 
-                <% if ruby_version(">= 2.7.0") %>
                   module_eval("def notify_admin(...); end")
```

Comment:
A bit nitpicky, but we shouldn't need the `module_eval` here anymore either, since I believe it was there so that Sorbet would not freak out or something:
```suggestion
                  def notify_admin(...); end
```